src/rollbar.js

```diff
@@ -164,6 +171,26 @@ Rollbar.prototype._log = function (defaultLevel, item) {
   }
 };
 
+Rollbar.prototype._addTracingAttributes = function (item) {
+  if (this.tracing) {
+    const span = this.tracing.getSpan();
+    if (!span) {
+      return;
+    }
```

Comment:
Could take advantage of this.tracing being null and just do:
```suggestion
  const span = this.tracing?.getSpan();
  if (!span) return;
```
Not sure though.

##### non-blocking